core/quantity/src/Formatter/FormatEnums.ts

```diff
@@ -123,6 +125,19 @@ export enum ScientificType {
   ZeroNormalized,
 }
 
+/** required if type is ratio
+ * @beta */
+export enum RatioType {
```

Comment:
why dont we use string enums?
with that could we get rid of `parseRatioType` and `ratioTypeToString`?

Reply:
Fair point, I'm changing it. It's not consistent with all the other enums in this same file though. We should probably go and adjust them all in a separate PR @hl662

Reply:
agreed, I'll have @franklbh tackle this, will create a separate issue